### DISCUSSION TITLE
Make rebase the default "Update branch" button

### DISCUSSION THREAD

**Discussion by guitaripod** [community/community#31994] [Pull Requests]
Is there a way to configure this currently?

I want the rebase function to be the default.

<img width="328" alt="CleanShot 2022-09-05 at 10 45 28@2x" src="https://user-images.githubusercontent.com/47460844/188396558-31a496e7-fdb2-484c-a018-d3ba23c79a2b.png">

**Comment by Bisaloo** [comment]
Duplicate of https://github.com/orgs/community/discussions/12032?
